haproxy22.spec

```diff
@@ -23,13 +21,8 @@ Source3:        haproxy.logrotate
 Source4:        haproxy.sysconfig
 Source5:        halog.1
 
-Patch0:         haproxy-2.2.0-lua-5.4.patch
-
 BuildRequires:  gcc
-%if %{with lua}
-# src/hlua.c: "Requires Lua 5.3 or later."
-BuildRequires:  lua-devel >= 5.3
-%endif
+BuildRequires:  lua53u-devel
```

Comment:
@carlwgeorge I see lua53u in the yum repo, but I don't see a corresponding github repo. Any ideas?

Reply:
It's still in the old org (https://github.com/iuscommunity-pkg/lua53u) and hasn't been migrated to https://github.com/iusrepo yet.  I was actually going to overhaul the package to submit it to EPEL instead, but never got around to it.